hail_search/queries/mito.py

```diff
@@ -135,7 +136,7 @@ def __init__(self, *args, **kwargs):
 
     def _parse_intervals(self, intervals, exclude_intervals=False, **kwargs):
         parsed_intervals = super()._parse_intervals(intervals,**kwargs)
-        if parsed_intervals and not exclude_intervals:
+        if parsed_intervals and not exclude_intervals and len(parsed_intervals) < MAX_LOAD_INTERVALS:
```

Comment:
is `load_table_kwargs` only passed to the annotations read table and not the entries read table?

Reply:
no its *only* passed to the entries read table. the annotations table is not read in, its just accessed via query_table